Will Heikoff Documentation

```diff
@@ -321,3 +321,49 @@ The assumption that the Zip Code Business Pattern Data is based off of USPS Zip
 	discretion of the U.S. Postal Service and may change from time to time”. This information can be found at the census business
 	patterns documentation website at https://www.census.gov/programs-surveys/cbp/technical-documentation/methodology.html
 
+7/31/18
+Created easier method of join CA_Block_2000.shp with the Block data that includes total employment without having to use access.
+	Did this by creating CA_WAC.csv file in excel (same as above on 7/17/18), then wrote script in R titled CA_WAC_Transformation.R,
+	of which imports the csv file as a table titled “Block”, takes away the scientific notation, and then exports the table a dbf 
+	file titled “CAL_Blocks” saved in the “G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\CARR\California\Base Data” file for use in 
+	ArcMap
+Imported CAL_Blocks.dbf data layer along with CA_Block_2000.shp data layer into ArcMap
+Opened CAL_Blocks.dbf data attribute table, selected “Add Field…” option from the “Table Options” drop-down menu. Named field “Block_ID”
+	and changed type to “Text” with Length set to 15. Right-clicked on newly created field and selected “Field Calculator…” from the
+	drop-down menu, changed Type to “String”, and entered “0&[W_GEOCODE]” and hit OK. This created the new “Block_ID” field with
+	California.xslx’s Block IDs with a 0 in front of each one, therefore matching up with CA_Block_2000.shp’s Block IDs.
```

Comment:
We may be able to skip this manual step too: see if you can transform this variable in R so that when you open it in ArcMap, it's already a string and no further manipulations have to be done. You'll have to Google to find the functions that do the equivalent variable transformations in R.